assignments/array-methods.js

```diff
@@ -58,28 +58,61 @@ const runners = [
 // ==== Challenge 1: Use .forEach() ====
 // The event director needs both the first and last names of each runner for their running bibs. Combine both the first and last names and populate a new array called `fullNames`. This array will contain just strings.
 let fullNames = [];
+  runners.forEach(runners => {
+  fullNames.push(`${runners.first_name} ${runners.last_name}`);
+  });
+    
 console.log(fullNames);
 
 // ==== Challenge 2: Use .map() ====
 // The event director needs to have all the runners' first names in uppercase because the director BECAME DRUNK WITH POWER. Populate an array called `firstNamesAllCaps`. This array will contain just strings.
 let firstNamesAllCaps = [];
+  runners.map((runners,index) => {
+    firstNamesAllCaps[index] = runners.first_name.toUpperCase();
+});
+
 console.log(firstNamesAllCaps);
 
 // ==== Challenge 3: Use .filter() ====
 // The large shirts won't be available for the event due to an ordering issue. We need a filtered version of the runners array, containing only those runners with large sized shirts so they can choose a different size. This will be an array of objects.
 let runnersLargeSizeShirt = [];
+runnersLargeSizeShirt = runners.filter(runner => 
+  runner.shirt_size === 'L');
+
```

Comment:
Since you're immediately assigning a value to the variable, you can avoid declaring it as an empty array first and assign it directly. 

`let runnersLargeSizeShirt = runners.filter(runner => runner.shirt_size === 'L');`